doc/threat-model.md

```diff
@@ -0,0 +1,9 @@
+Until there is a more concise explanation here about the threat model (what it
+defends against) and assurance case (why it is secure), one should be able to
+get the necessary informaion from this blog post:
+https://cockpit-project.org/blog/is-cockpit-secure.html
```

Comment:
This paragraph is a placeholder, and the blog post is rather dated -- the screenshots don't match current versions any more, and the video is gone. The text is still current, though. Perhaps we should copy the text here then? But that's still not the format of a threat model, "just" a description.